### ISSUE TITLE
Failed to import into Notepad++ v7.3.1 (32-bit)

### ISSUE THREAD

**Issue by beJels** [WorksOfBarry/Notepad-RPG#1]
Hi,

I tried importing the sqlrpgle.xml into Notepad++, but it failed.
It says: "Fail to import."
Any idea?

**Comment by WorksOfBarry** [comment]
Hi @beJels 

Make sure you imported the correct file.

The error is a big vague, so unless you provide some more info I can't do much else for now.

Liam 😄

**Comment by beJels** [comment]
I've got it figured out!
I saved the wrong file from my browser.

WRONG APPROACH: In my browser I right clicked "sqlrpgle.xml" and chose "save target as", in the dialog window google chrome browser proposed the filename "sqlrpgle.xml", therefore I thought I was saving the actual xml.
But it turned out I was downloading the html page.

RIGHT APPROACH: In the browser I should have just clicked on "sqlrpgle.xml", this shows the file in the gitHub website. Right there I can right click the "Raw" button, which in the dialog window proposes me again the filename "sqlrpgle.xml". This time it is actually the xml.
